.python-version

```diff
@@ -1 +1 @@
-3.11
+3.10
```

Comment:
we use python 3.11 in `simcore`, `osparc-config` and `osparc-infra` repositores. We also use it on our server machines (check [this Issue](https://github.com/ITISFoundation/osparc-ops-environments/issues/883)).

Is there a specific reason to use 3.10 in `osparc-ops`?

Reply:
Good info thanks, I think something didnt work with 3.11 and I simply reverted to 3.10. But in this case it is a good moment to fix the issues and move to 3.11. I will have another look ;)

Reply:
3.11 works here just fine, going back to 3.11 :--) thanks!